tests/lax_numpy_test.py

```diff
@@ -2652,7 +2652,7 @@ def testSort(self, dtype, shape, axis):
       {"testcase_name": "_{}_axis={}".format(
           jtu.format_shape_dtype_string(shape, dtype), axis),
         "shape": shape, "dtype": dtype, "axis": axis}
-      for dtype in all_dtypes
+      for dtype in python_scalar_dtypes
```

Comment:
Please change this back to `all_dtypes` – we want to test more than just python scalar types.